### PR TITLE
Show command to run upload if should_upload had been false

### PR DESCRIPTION
**Pull request by aylusltd**
## Checks

- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by SeanTConrad — APPROVED** [review]

File: src/verinfast/agent.py [hunk 1]
```diff
@@ -856,6 +860,11 @@ def main():
         print(f"""The log for this run has moved to:
               {d}/{new_folder_name}/{new_file_name}""")
 
+    # We only do this if you have a remote config but didn't upload
+    if agent.config.shouldUpload is False and agent.config.is_path_remote():
```
Comment (SeanTConrad) [inline src/verinfast/agent.py:864]: is_path_remote() won't work. config.cfg_path gets overwritten from URL to local path

File: src/verinfast/agent.py [hunk 2]
```diff
@@ -856,6 +860,11 @@ def main():
...
+        print("To upload results from this location please run")
```
Comment (SeanTConrad) [inline src/verinfast/agent.py:865]: Minor: Can we put a few newlines in the beginning and change it to "To upload results from this scan, please run this command:"

**Review by SeanTConrad — CHANGES_REQUESTED** [review]

File: src/verinfast/agent.py (hunk 1 above)
Comment (SeanTConrad) [reply src/verinfast/agent.py]: I went ahead and fixed this.

File: src/verinfast/agent.py (hunk 2 above)
Comment (SeanTConrad) [reply src/verinfast/agent.py]: I fixed this.

File: src/verinfast/config.py [hunk 3]
```diff
@@ -398,7 +400,7 @@ def handle_args(self, args):
                 self.truncate_findings = False
 
     def is_path_remote(self) -> bool:
```
Comment (aylusltd) [inline src/verinfast/config.py:402]: we should rename the function too to avoid confusion.

Comment (SeanTConrad) [reply src/verinfast/config.py]: fixed

**Review by SeanTConrad — APPROVED** [review]

**Pull request merged**